doc/Language/faq.pod6

```diff
@@ -370,6 +370,99 @@ L<.comb|/routine/comb> is accepting an optional L<Int|/type/Int>:
 .say for 'abcdefghijklmnopqrstuvwxyz'.comb: 8;  # OUTPUT: «abcdefgh␤ijklmnop␤qrstuvwx␤yz»
 =end code
 
+=head2 List: How can I remove duplicates items in a list
+
+Use L<.unique|/routine/unique>:
+
+=begin code
+(1, 2, 3, 2, 3, 1, 1, 0).unique;  # OUTPUT: «(1 2 3 0)␤»
+=end code
+
+=head2 List: How can I delete an element from a list
+
+To delete by index, use L<.splice|/routine/splice>:
+
+=begin code
+my @a = <3 18 4 8 92 14 30>;
+say @a.splice(3, 1);  # OUTPUT: «[8]␤»
+say @a;               # OUTPUT: «[3 18 4 92 14 30]␤»
+=end code
+
+Note that the L<delete subscript|/language/subscripts#index-entry-:delete_(subscript_adverb)> replaces the element with its natural (i.e. default) state. Here (Any):
+
+=begin code
+my @a = <3 18 4 8 92 14 30>;
+@a[3]:delete;
+say @a;  # OUTPUT: «[3 18 4 (Any) 92 14 30]␤»
+=end code
+
+To delete by value, use L<.grep|/routine/grep>:
+
+=begin code
+my @letters = <a b c>;
+@letters .= grep: * ne "b";
+say @letters;  # OUTPUT: «[a c]␤»
+=end code
+
+=head2 List: How can I show index while looping through a list
+
+Use L<.kv|/routine/kv> or L<.pairs|/routine/pairs>:
+
+=begin code
+my @aa = 8 .. 12;
+for @aa.kv -> $i, $_ { say "$i: $_" };     # OUTPUT: «0: 8␤1: 9␤2: 10␤3: 11␤4: 12␤»
+say "$_.key(): $_.value()" for @aa.pairs;  # OUTPUT: «0: 8␤1: 9␤2: 10␤3: 11␤4: 12␤»
+=end code
+
+=head2 List: How can I concatenate lists
+
+Use the slip prefix (L<C<|>|/language/operators#prefix_|>) or the L<flat|/routine/flat> routine
+
+=begin code
+my @a = <a b c>;
+my @b = <d e f>;
+my @ab = |@a, |@b; say @ab;  # OUTPUT: «[a b c d e f]␤»
+say (@a, @b).flat;           # OUTPUT: «(a b c d e f)␤»
+=end code
+
+Note that flat recursively flattens L<non-containerized|/language/containers> L<iterables|/type/Iterable>.
+
+=head2 List: How can I generate a list of random numbers
+
+Use L<.roll|/routine/roll>:
+
+=begin code
+say (1..100).roll(100);
+=end code
+
+Prints 50 numbers between 1 and 100.
+
```

Comment:
It prints 100. Also, that's a simple example of using roll. Just add another example to roll.

Reply:
Added:

lazy example with `*` ([strangely consistent](http://strangelyconsistent.org/blog/6-builtins-in-perl-6-that-you-never-knew-you-needed) has also some nice one)

For your information:
What I was looking for actually with this question is that making a list of 100M random integer is faster than generating 100M random integers. But this time is finished apparently because the C++ libs are keeping the state and optimization results in the same.
So the only clever performance trick is with [RDRAND](https://stackoverflow.com/questions/10484164) with [benchmark](https://stackoverflow.com/questions/26771329).